code/game/objects/effects/phased_mob.dm

```diff
@@ -10,7 +10,9 @@
 
 /obj/effect/dummy/phased_mob/Destroy()
 	// Eject contents if deleted somehow
-	var/area/destination_area = get_turf(src)
+	var/atom/drop_to = drop_location()
+	var/turf/just_in_case = get_turf(drop_to)
+	var/area/destination_area = just_in_case.loc
```

Comment:
Mobile posting but We have get_area if you don’t need to otherwise use the turf.

Reply:
👍